Keycloak.Migrator/Extensions/SynchronizeCommandExtension.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Autofac;
+using Keycloak.Migrator.DataServices.Interfaces;
+using Keycloak.Migrator.Models;
+using System;
+using System.CommandLine;
+using System.IO;
+using System.Linq;
+
+namespace Keycloak.Migrator.Extensions
+{
+    internal static class SynchronizeCommandExtension
+    {
+        public static Command AddSynchronizeCommand(this Command command)
+        {
+
+            Option<Uri> keycloakUri = new Option<Uri>(
+               "--keycloak-url",
+               "The Uri to the Keycloak Endpoint."
+            )
+            {
+                IsRequired = true,
+            };
+
+            Option<string> keycloakClientId = new Option<string>(
+               "--keycloak-client-id",
+               "The id of the client in Keycloak to sync."
+)
+            {
+                IsRequired = true,
+            };
+
+            Option<string> keycloakPassword = new Option<string>(
+                "--keycloak-password",
+                "The password to the Keycloak Endpoint."
+            )
+            {
+                IsRequired = true,
+            }; ;
+
+            Option<string> keycloakUserName = new Option<string>(
+                "--keycloak-username",
+                "The user name to the Keycloak Endpoint."
+            )
+            {
+                IsRequired = true,
+            };
+
+            Option<bool> deleteMissingUsers = new Option<bool>(
+                "--delete-missing-users",
+                "If flag is set, delete users that are in the realm but not the export"
```

Comment:
Let's specify the default value for the option.

```suggestion
            Option<bool> deleteMissingUsers = new Option<bool>(
                "--delete-missing-users",
                () => false,
                "If flag is set, delete users that are in the realm but not the export"
            )
            {
                IsRequired = false,
            };
```

Reply:
Thanks! I tried to find a default value as a property but didn't realize it was another constructor argument.